utils/get_supported_scylla_base_versions.py

```diff
@@ -15,7 +15,7 @@
 
 
 # We support to migrate from specific OSS version to enterprise
-supported_src_oss = {'2021.1': '4.3', '2020.1': '4.0', '2019.1': '3.0'}
+supported_src_oss = {'2022.1': '4.6', '2021.1': '4.3', '2020.1': '4.0', '2019.1': '3.0'}
```

Comment:
Now it makes me wonder what is the base version for 2022.1.
In all previous enterprise versions it was the version we branched from, so in this case it should have been 5.0.
However, since we're working on them together maybe it's different now.
Let's test with 4.6 and see how it goes.